### ISSUE TITLE
Android static App shortcuts

### ISSUE THREAD

**Issue by Peretz30** [jordanbyron/react-native-quick-actions#98]
For now library supports only dynamic shortcuts for android. Any plans about static ones?
@huangmr

**Comment by badaz** [comment]
Is there a plan to merge the PR someday? Looks like it has been around since May 2020. Looks like the last commits to the master branch of this repo go back to May 2020 too. Is the project still maintained?

**Comment by jordanbyron** [comment]
> Is the project still maintained?

Not really. Without a test suite to rely on, and my lack of day-to-day use of react-native, I have not felt comfortable evaluating and merging new PRs to this project. I'd happily unlock commit access to anyone wanting to take up the task.
